extern/ttconv/pprdrv_tt2.cpp

```diff
@@ -89,12 +89,6 @@ struct FlaggedPoint
     FlaggedPoint(Flag flag_, FWord x_, FWord y_): flag(flag_), x(x_), y(y_) {};
 };
 
-double area(FWord *x, FWord *y, int n);
-#define sqr(x) ((x)*(x))
-
-#define NOMOREINCTR -1
-#define NOMOREOUTCTR -1
-
```

Comment:
Why is this part of the PR? This does not have anything to do with DEBUG_TRUETYPE.

Reply:
It's the "& removals of unused defs." part.

Reply:
:sheep: